### PR TITLE
Exception created

### PR DESCRIPTION
**Pull request by zuhriddinzayniddinov**

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (iCloudBMX) [inline .vscode/tasks.json]: bu folderni ignore qiling

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (iCloudBMX) [inline .vscode/launch.json]: Bu folderni ignore qiling

File: src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs [hunk 3]
```diff
@@ -45,6 +46,46 @@ public async Task HandleCommandAsync(Update update)
 
             await task;
         }
+        catch(AlreadyExictException exception)
```
Comment (iCloudBMX) [inline src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs]: AlreadyExistsException bo'lishi kerak name

File: src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs [hunk 4]
```diff
@@ -45,6 +46,46 @@ public async Task HandleCommandAsync(Update update)
...
+        {
+            this.logger.LogError(exception.Message);
+
+            await this.telegramBotClient.SendTextMessageAsync(
+                chatId: message.From.Id,
+                text: exception.Message);
+
+            return;
+        }
+        catch(NotFaundLeetCodeUserNameException exception)
```
Comment (iCloudBMX) [inline src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs]: NotFoundException bo'lgani yaxshi

File: src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs [hunk 5]
```diff
@@ -45,6 +46,46 @@ public async Task HandleCommandAsync(Update update)
...
+        {
+            this.logger.LogError(exception.Message);
+
+            await this.telegramBotClient.SendTextMessageAsync(
+                chatId: message.From.Id,
+                text: exception.Message);
+
+            return;
+        }
+        catch(DuplicateTelegramIdException exception)
```
Comment (iCloudBMX) [inline src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs]: DuplicateUserException

File: src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs [hunk 6]
```diff
@@ -45,6 +46,46 @@ public async Task HandleCommandAsync(Update update)
...
+        {
+            this.logger.LogError(exception.Message);
+
+            await this.telegramBotClient.SendTextMessageAsync(
+                chatId: message.From.Id,
+                text: exception.Message);
+
+            return;
+        }
+        catch(DuplicateLeetCodeUserNameException exception)
```
Comment (iCloudBMX) [inline src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs]: Tepadagi exception yetadi

Comment (iCloudBMX) [reply src/DailyLeetcodeReminder.Core/Services/CommandHandler.cs]: Exception larni user'ga yubormang bu xatolik, o'sha xatolikka mos xabar yuboring, error'lar saqlavchi birorta resource file tashkil qiling

File: src/DailyLeetcodeReminder.Domain/Exceptions/AlreadyExict.cs [hunk 7]
```diff
@@ -0,0 +1,8 @@
+namespace DailyLeetcodeReminder.Domain.Exceptions;
+
+public class AlreadyExictException : Exception
+{
+  public AlreadyExictException() :
```
Comment (iCloudBMX) [inline src/DailyLeetcodeReminder.Domain/Exceptions/AlreadyExict.cs]: Tepadagi muammolarni ko'rib chiqing

**Review by iCloudBMX — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (zuhriddinzayniddinov) [reply .vscode/tasks.json]: Adashib ketib qolibti

**Review by zuhriddinzayniddinov — COMMENTED** [review]
I haven't tried running, but I think the logic is correct

**Review by iCloudBMX — APPROVED** [review]

**Pull request merged**